wordpress-mobile/WordPress-iOS
# Empty Stats: remove AB testing

This PR removes the AB testing from Empty Stats based on results from pbArwn-3ce-p2.

### To test

1. Do a fresh install
2. Select a site with 30 or fewer visitors
3. Make sure that the first nudge is social/publicize ("A tip to grow your audience")
4. Make sure the events don't have the `firstNudge` property

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
